cmd/miniooni/main.go

```diff
@@ -74,11 +74,12 @@ func init() {
 		"Set collector base URL", "URL",
 	)
 	getopt.FlagLong(
-		&globalOptions.extraOptions, "option", 'O',
-		"Pass an option to the experiment", "KEY=VALUE",
+		&globalOptions.inputs, "input", 'i',
+		"Add test-dependent input to the test input", "INPUT",
 	)
 	getopt.FlagLong(
-		&globalOptions.logfile, "logfile", 'l', "Set the logfile path", "PATH",
+		&globalOptions.extraOptions, "option", 'O',
+		"Pass an option to the experiment", "KEY=VALUE",
```

Comment:
We should try to avoid these artefacts. Also, `extraOptions` comes before alphabetically anyway. Since what we're doing here is just adding the `inputs` flag, via the master branch, let's do this in a way in which we do not create unnecessary noise in the diff. Basically, I am suggesting to move `extraOptions` above `inputs` such that it would not be actually interested by the diff.

Reply:
This lines has been deleted. I wasn't to blame.

Reply:
(I was not blaming)